__tests__/users.ts

```diff
@@ -2412,6 +2414,29 @@ describe('mutation subscribePersonalizedDigest', () => {
       preferredTimezone: 'Europe/Athens',
     });
   });
+
+  it('should subscribe to reading reminder', async () => {
+    loggedUser = '1';
+
+    const res = await client.mutate(MUTATION, {
+      variables: {
+        type: UserPersonalizedDigestType.ReadingReminder,
+      },
+    });
+    expect(res.errors).toBeFalsy();
+    expect(res.data.subscribePersonalizedDigest).toMatchObject({
+      preferredDay: DayOfWeek.Monday,
+      preferredHour: 9,
+      preferredTimezone: 'Etc/UTC',
+    });
```

Comment:
Think you need type here now as well :)

Reply:
passes locally

Reply:
`toMatchObject` does not test extra values on object, but yeah, we can probably add to be sure it is correct 👍

Reply:
Yeah this was what I ment :) 
We would have seen it failed as it was not added to object.

Maybe you meant `toStrictEqual`

Reply:
That one would be better agreed 👍